--- ../../../app/messages/confirm_registration.email.html.erb	2020-12-16 22:03:48.796101467 +0000
+++ messages/confirm_registration.email.html.erb	2021-02-12 22:18:16.456387907 +0000
@@ -1,6 +1,6 @@
 <% p = asset.is_a?(Pseudonym) ? asset : (asset.pseudonym || asset.user.pseudonym) %>
 <% a = p.account %>
-<% product_name = a.product_name || "Canvas" %>
+<% product_name = a.product_name || "Lantern" %>
 <% cc = asset.is_a?(CommunicationChannel) ? asset : (p.communication_channel || p.user.communication_channel) %>
 
 <% define_content :link do %>
