--- ../../../app/messages/forgot_password.email.html.erb	2020-10-27 05:38:02.936594089 +0000
+++ messages/forgot_password.email.html.erb	2020-11-02 18:32:24.291124818 +0000
@@ -3,10 +3,10 @@
 <% end %>
 
 <% define_content :subject do %>
-  <%= t('subject', 'Forgot Password: Canvas') %>
+  <%= t('subject', 'Forgot Password: Lantern') %>
 <% end %>
 
-<p><%= t('requested_password_reset', 'You requested a confirmation of your password for logging into Canvas.') %></p>
+<p><%= t('requested_password_reset', 'You requested a confirmation of your password for logging into Lantern.') %></p>
 
 <% pseudonyms = asset.user.all_active_pseudonyms %>
 <% first_pseudonym = pseudonyms.first %>
