--- ../../../app/messages/summaries.email.html.erb	2020-10-27 05:38:02.949927545 +0000
+++ messages/summaries.email.html.erb	2021-02-12 22:18:24.213143717 +0000
@@ -3,7 +3,7 @@
 <% end %>
 
 <% define_content :subject do %>
-  <%= t :subject, "Recent Canvas Notifications" %>
+  <%= t :subject, "Recent Lantern Notifications" %>
 <% end %>
 
 <% define_content :report_date do %>
@@ -12,10 +12,10 @@
 
 <% m = (delayed_messages || []).find{|m| m.frequency == Notification::FREQ_WEEKLY} || delayed_messages.first %>
 <% if !m || m.frequency == Notification::FREQ_DAILY %>
-  <p><%= t :daily_body, "You're signed up to receive a daily report of some notifications from your Canvas account.  Below is the report for %{date}:",
+  <p><%= t :daily_body, "You're signed up to receive a daily report of some notifications from your Lantern account.  Below is the report for %{date}:",
            :date => content(:report_date) %></p>
 <% elsif m.frequency == Notification::FREQ_WEEKLY %>
-  <p><%= t :weekly_body, "You're signed up to receive a weekly report of some notifications from your Canvas account.  Below is the report for the week ending %{date}:",
+  <p><%= t :weekly_body, "You're signed up to receive a weekly report of some notifications from your Lantern account.  Below is the report for the week ending %{date}:",
            :date => content(:report_date) %></p>
 <% end %>
 
